app/javascript/Pages/AvatarUpload.jsx

```diff
@@ -23,17 +19,17 @@ export default function Profile(props) {
     state.avatar = event.target.files[0];
   };
 
-  const token = document.querySelector(
-    "[name=csrf-token]"
-  ).content;
+  const token = document.querySelector("[name=csrf-token]").content;
```

Comment:
**chore (blocking)**: don't forget to update these files with the new window variable